gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -1187,29 +1203,35 @@ def main(args):
                     variant_filter_field="AS_VQSR",
                     problematic_regions=REGION_FLAG_FIELDS[data_type],
                     single_filter_count=True,
-                    filters_check=False if data_type == "joint" else True,
+                    filters_check=not for_joint,
                 )
-                dt_ht.describe()
-
-                if data_type == "joint":
+                if for_joint:
                     no_dt = ["region_flags", "allele_info", "freq_comparison_stats"]
                     dt_ht = dt_ht.rename(
                         {f: f"{dt}_{f}" for f in dt_ht.row_value if f not in no_dt}
                     )
-
+                    dt_ht = dt_ht.transmute_globals(
```

Comment:
It doesn't matter too much since the joint validated ht is not actually used for anything. I debated dropping the write of the final joint validated table but figured Id keep it in for now in case people could think of reasons to keep it.